packages/libs/chainweb-node-client/src/poll.ts

```diff
@@ -17,10 +17,16 @@ import { stringifyAndMakePOSTRequest } from './stringifyAndMakePOSTRequest';
 export async function poll(
   requestBody: IPollRequestBody,
   apiHost: string,
+  confirmationDepth = 0,
```

Comment:
we should probably add it to 3 or 5, I forgot what the common consensus is on this. @Takadenoshi ?

Reply:
What is the default value in the server?
What if I just make this optional